.github/workflows/testonpush.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       id-token: write
       contents: read
       checks: write
-      issues: read
+      pull-requests: write
```

Comment:
That's great, as you say, it's what specified in https://github.com/EnricoMi/publish-unit-test-result-action?tab=readme-ov-file#permissions !

```
permissions:
  checks: write
  pull-requests: write
```